src/compo/pandora_2ioda.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+import sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+import os
+import re
+from pathlib import Path
+
+import xarray as xr
+import math
+from numpy import log as ln
+
+import pyiodaconv.ioda_conv_engines as iconv
+from collections import defaultdict, OrderedDict
+from pyiodaconv.orddicts import DefaultOrderedDict
+from pyiodaconv.def_jedi_utils import epoch, iso8601_string
+
+# constants
+MOLECCM_2MOLM_2 = 10000/6.0221408e+23  # molec/cm2 to mol/m2
```

Comment:
optional but:
```
import scipy.constants as sc

avogadro_constant = sc.N_A
```

Reply:
Obs is already in `mol/m2`, I removed this line.